### PR TITLE
new: New landing page for docs

### LINKED ISSUE #5089: Docs landing page and navigation
- revamp landing page for docs
- review navbar

https://www.notion.so/prismaio/Home-for-PDP-Docs-Q2-2023-d06bf035929b49d2850f1b24e55f262d 

Design: https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0

### PR DESCRIPTION
**Pull request by carlagn**
## Describe this PR
revamp landing page for docs
https://www.notion.so/prismaio/Home-for-PDP-Docs-Q2-2023-d06bf035929b49d2850f1b24e55f262d

Design: https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0

## Changes

- Update header/footer/index page
- Add missing technology svgs

## What issue does this fix?

Fixes #5089 

## Any other relevant information

Slack feedback thread: https://prisma-company.slack.com/archives/C5Z9TH6N9/p1690816036118739

### CODE REVIEW

**Review by nilubava — APPROVED** [review]

**Comment by nilubava** [comment]
Suggestions / feedback from https://prisma-company.slack.com/archives/C5Z9TH6N9/p1690816036118739

- [x] The github icon doesn't appear to be a link.
- [x] "cloud projects" and "classic projects" links seem broken
- [x] font jitter when the page loads
- [x] Prisma logo should link to [prisma.io](http://prisma.io/) I think
- [x] "Docs" links to 404
- [x] The tiny house in the breadcrumb also links to 404
- [x] No page title
- [x] "Example projects" is an external link, but looks the same as internal one
- [x] Some links (towards bottom) are underlined
- [x] Kerning of "databases" buttons looks a bit off
- [x] https://pagespeed.web.dev/analysis/https-docs-git-new-docs-landing-page-prisma-vercel-app/5gsv4at8x0?form_factor=mobile + https://pagespeed.web.dev/analysis/https-docs-git-new-docs-landing-page-prisma-vercel-app/5gsv4at8x0?form_factor=desktop probably deserves a look
- [x] Quite a few 404s for fonts that don't exist: https://docs-git-new-docs-landing-page-prisma.vercel.app/docs/fonts/Inter-Regular.woff2
- [x] The "404" page does not seem to be centered
- [x] og:url is invalid for all pages
- [x] The left hand sidebar lost the "headline" (Get started, Concepts) etc - is that intentional?
- [x] Pages with long right-hand navigations now have a slightly different scrolling behavior: https://docs-git-new-docs-landing-page-prisma.vercel.app/reference/api-reference/prisma-client-reference vs. https://www.prisma.io/docs/reference/api-reference/prisma-client-reference
- [x] Less vertical content visible as the header always sticks around

**Comment by nilubava** [comment]
> "cloud projects" and "classic projects" links seem broken 

 @nikolasburk / @janpio Could you help me with the correct links here?

> Some links (towards bottom) are underlined

@janpio This is according to design in https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0

> og:url is invalid for all pages

This issue happens only in previews - We have a blanket issue for fixing this across all the repos soon

>  Less vertical content visible as the header always sticks around

@janpio Looks like this is something according to agreed upon design interactions. Do you have any suggestions for this?

**Comment by janpio** [comment]
> > Some links (towards bottom) are underlined
> 
> @janpio This is according to design in [figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0](https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0)

Ok, we should refine the design then to be consistent.


> > Less vertical content visible as the header always sticks around
> 
> @janpio Looks like this is something according to agreed upon design interactions. Do you have any suggestions for this?

Same here, I can give feedback that now less content is visible and that makes it harder to use the docs site and read its content. Potentially this design decision should be revisited.

**Comment by nilubava** [comment]
More feedback:

- [x] bullet points are not visible in lists in safari, although they are in firefox
- [x] still have the flash of everything (it loads, then disappears for a few ms, and appears again).
- [x] "Planetscale" button should probably link to https://docs-git-new-docs-landing-page-prisma.vercel.app/guides/database/planetscale instead of [prisma.io/planetscale](http://prisma.io/planetscale)
- [x] https://docs-git-new-docs-landing-page-prisma.vercel.app/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client vs. https://docs-git-new-docs-landing-page-prisma.vercel.app/concepts/components/prisma-client/working-with-prismaclient/connection-management - note how the content area shrinks in the second URL
Also happening for https://docs-git-new-docs-landing-page-prisma.vercel.app/concepts/components/prisma-client/full-text-search and https://docs-git-new-docs-landing-page-prisma.vercel.app/concepts/components/prisma-client/custom-validation)
- [x] From @nikolasburk : Tbh, I would probably not even link to Cloud Projects and Classic Projects from this landing page. These links are not crucial and I think they’d confuse people more than clarify things because by putting them there on the landing page we signal that these are “relevant”/“important” to look at — which I don’t think they are.
So I’d just omit Cloud Projects and Classic Projects from here

**Comment by Techboidesign** [comment]
> > > Some links (towards bottom) are underlined
> > 
> > 
> > @janpio This is according to design in [figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0](https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1565-2815&mode=design&t=f1yceOEYsGPwbtIM-0)
> 
> Ok, we should refine the design then to be consistent.
> 
> > > Less vertical content visible as the header always sticks around
> > 
> > 
> > @janpio Looks like this is something according to agreed upon design interactions. Do you have any suggestions for this?
> 
> Same here, I can give feedback that now less content is visible and that makes it harder to use the docs site and read its content. Potentially this design decision should be revisited.

The **navbar** is the only element supposed to be **sticky** here, not the search bar! I've added a **comment directly on the PR for this** @janpio @nilubava

**Comment by Techboidesign** [comment]
@nilubava I added a couple of comments on the **PR** directly. 

- [x] All links should have **underline** as a hover state
- [x] **Search bar** in the **homepage** should **NOT** be sticky :) 
- [x] The **three lower cards** at the end of the page now have _no underline_ as default, to be consistent with the rest

**Comment by nilubava** [comment]
- [ ] Search bar in the mobile version is not displayed properly

**Comment by Techboidesign** [comment]
@nilubava 

- [x] all these also **need underline when hovering**

image

**Comment by Techboidesign** [comment]
also @nilubava 

- [x] the **menu HOVER state is different** should be like this: 

image

**Comment by Techboidesign** [comment]
also @nilubava

- [x]  in some sections the **breadcrumbs** get cropped and the **line height is minimal**

image

- [x] Also these **two drop-downs** should go **below the breadcrumbs**

image

**Comment by Techboidesign** [comment]
@nilubava 

- [x] The **search bar** should NOT have a _flat background_ beneath, but just a drop shadow, like in the **[Figma Prototype](https://www.figma.com/file/aTK3UuDkVlld0G1q0ik9fw/Docs-Website?type=design&node-id=1573-4948&mode=design&t=qUeXVE7VnIERavfW-11)**  - also notice the border radius + color
image
image

**Comment by Techboidesign** [comment]
@carlagn there still a page that does some funky-width things - **all pages should have the same width**
https://docs-git-new-docs-landing-page-prisma.vercel.app/reference/database-reference/database-features
image

Probably because of these tables, in the middle they have some funny spacing: 
image

You can play with the **max-width parameter** and **font-size too** (I think other pages have a **668px width**)
image

File: config.ts [hunk 1]
```diff
@@ -76,328 +76,151 @@ const siteConfig = {
...
         ],
       },
-      {
-        categoryName: 'Components',
-        icon: 'ComponentsIcon',
+      pdp: {
+        title: 'Prisma Data Platform',
+        icon: 'triangle',
+        description:
+          'An ecosystem of tools to empower teams to easily create and launch data-heavy, global-first software.',
```
Comment (luanvdw) [inline config.ts]:
```suggestion
          'An ecosystem of tools empowering teams to build data-heavy, global-first applications'
          'An ecosystem of tools to empower teams to easily create and launch data-heavy, global-first software.',
```

File: config.ts [hunk 2]
```diff
@@ -76,328 +76,151 @@ const siteConfig = {
...
-            text: 'Prisma CLI',
-            url: 'concepts/components/prisma-cli',
-            codeBlock: false,
-          },
-          {
-            text: 'Prisma Studio',
-            url: 'concepts/components/prisma-studio',
-            codeBlock: false,
+            url: 'data-platform/accelerate',
+            title: 'Accelerate',
```
Comment (luanvdw) [inline config.ts]:
```suggestion
            title: 'Prisma Accelerate',
```

File: config.ts [hunk 3]
```diff
@@ -76,328 +76,151 @@ const siteConfig = {
...
-            text: 'SQLite',
-            url: 'concepts/database-connectors/sqlite',
-            codeBlock: false,
-          },
-          {
-            text: 'CockroachDB',
-            url: 'concepts/database-connectors/cockroachdb',
-            codeBlock: false,
+            url: 'data-platform/pulse',
+            title: 'Pulse',
```
Comment (luanvdw) [inline config.ts]:
```suggestion
            title: 'Prisma Pulse',
```

**Comment by meletj** [comment]
two more fixes; 
- [x] can we make the Github logo in navbar open the link in new tab? 
- [x] when clicking the 'Docs' text in nav bar, it goes to 404 instead of back to the Docs homepage?

**Comment by nilubava** [comment]
@meletj 
> when clicking the 'Docs' text in nav bar, it goes to 404 instead of back to the Docs homepage?


This is because our previews do not have the /docs prefix. But prefixes work in production, as you can see the behaviour of the current '/docs' link in our navbar. We could test this by enabling the prefix for this branch

**Comment by meletj** [comment]
> @meletj
> 
> > when clicking the 'Docs' text in nav bar, it goes to 404 instead of back to the Docs homepage?
> 
> This is because our previews do not have the /docs prefix. But prefixes work in production, as you can see the behaviour of the current '/docs' link in our navbar. We could test this by enabling the prefix for this branch

All good then

**Review by meletj — APPROVED** [review]
All changes requested in Slack have been implemented. Copy and links approved.

**Pull request merged**